cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -131,7 +142,10 @@ def download_data(dataset: str) -> FlyteDirectory:
     # return the directory populated with rossmann data files
     return FlyteDirectory(path=str(data_dir))
 
-
+# %%
```

Comment:
Can we have a lead-in to explain what's gonna be done in the next few functions (e.g., let's perform data pre-processing now!)? Every content chunk has to be like a story! :)

Reply:
Sure, this statement was already there from before. I thought I would talk to you before discussing it. Yes, I will do that

Reply:
Here we can say, "First, we clean and prepare the Google trend data". (let's just simplify the content).